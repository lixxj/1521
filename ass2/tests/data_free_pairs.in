a = malloc 40
b = malloc 100
c = malloc 40
d = malloc 100
free a
free c
a = malloc 20
c = malloc 20
free c
free a