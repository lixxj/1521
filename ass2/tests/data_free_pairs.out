+00000 (F, 5000) 
[a] +00008 
+00000 (A,   48) +00048 (F, 4952) 
[a] +00008 [b] +00056 
+00000 (A,   48) +00048 (A,  108) +00156 (F, 4844) 
[a] +00008 [b] +00056 [c] +00164 
+00000 (A,   48) +00048 (A,  108) +00156 (A,   48) +00204 (F, 4796) 
[a] +00008 [b] +00056 [c] +00164 [d] +00212 
+00000 (A,   48) +00048 (A,  108) +00156 (A,   48) +00204 (A,  108) +00312 (F, 4688)
[b] +00056 [c] +00164 [d] +00212 
+00000 (F,   48) +00048 (A,  108) +00156 (A,   48) +00204 (A,  108) +00312 (F, 4688)
[b] +00056 [d] +00212 
+00000 (F,   48) +00048 (A,  108) +00156 (F,   48) +00204 (A,  108) +00312 (F, 4688)
[a] +00008 [b] +00056 [d] +00212 
+00000 (A,   48) +00048 (A,  108) +00156 (F,   48) +00204 (A,  108) +00312 (F, 4688)
[a] +00008 [b] +00056 [c] +00164 [d] +00212 
+00000 (A,   48) +00048 (A,  108) +00156 (A,   48) +00204 (A,  108) +00312 (F, 4688)
[a] +00008 [b] +00056 [d] +00212 
+00000 (A,   48) +00048 (A,  108) +00156 (F,   48) +00204 (A,  108) +00312 (F, 4688)
[b] +00056 [d] +00212 
+00000 (F,   48) +00048 (A,  108) +00156 (F,   48) +00204 (A,  108) +00312 (F, 4688)
+00000 (F,   48) +00048 (A,  108) +00156 (F,   48) +00204 (A,  108) +00312 (F, 4688)
